Give examples of pointer event handling. (#1224)
MicroEJ/docs: ApplicationDeveloperGuide/UI/MWT/concepts.rst

   :alt: Pointer Event Dispatcher Flow
   :align: center

Use case examples:

- A button-like widget should not consume the press event since the user can drag the pointer after that.

  - In a simple case, the user may release the pointer outside the bounds of the button, thus the button will not be clicked.
  - Or the button may be included in a container that capture the drag events (and/or the release ones) to perform a scroll or a drag-and-drop for instance.

- In the same way, a directed scroll (horizontal or vertical) should only start to consume pointer events once a swipe in the right direction is detected.

>   This way, 2 scrolls with a different direction may be nested in one another, each one handling one direction.

Reviewer: ```suggestion
  This way, 2 scrolls with different directions may be nested, each one handling one direction.
```

Reviewer: Suggestion applied.
